stats/statsreporter.go

```diff
@@ -60,18 +61,20 @@ type Sample struct {
 // the data to the TSDB. Stops when close signal is received on closeChannel.
 func (sr *statsReporter) report(d time.Duration) {
 	tc := time.Tick(d)
-	plog.Infof("collecting internal metrics at %s intervals", d)
+	plog.WithField("interval", d).Info("Collecting internal metrics")
```

Comment:
Don't log info level at the beginning. Info is a measurable unit of positive work. "Internal metrics collected" would be okay after it succeeds, but even then it's common enough that it should be debug